src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/arm/conv_fq.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/node_builders/convolution.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "utils/fusing_test_utils.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/fake_quantize.hpp"
+#include "openvino/op/matmul.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+class ConvAndFQ : virtual public SubgraphBaseTest, public CPUTestsBase {
+public:
+    void SetUp() override {
+        abs_threshold = 4e-3f;
+        targetDevice = ov::test::utils::DEVICE_CPU;
+
+        std::tie(inFmts, outFmts, priority, selectedType) = CPUSpecificParams{{}, {}, {}, CPUTestsBase::any_type};
+        const auto precision = element::f32;
+        const auto input_static_shape = Shape{4, 3, 2, 2};
+
+        auto in_shapes = static_shapes_to_test_representation({input_static_shape});
+        init_input_shapes({in_shapes});
+        ov::ParameterVector input_params{
+            std::make_shared<ov::op::v0::Parameter>(precision, ov::Shape(input_static_shape))};
+
+        auto fq_before = ov::test::utils::make_fake_quantize(input_params[0], precision, 256, {}, {-1.28f}, {1.27f}, {-1.28f}, {1.27f});
+
+        auto weights_shape = Shape{4, 3, 2, 2};
+        auto weights = utils::make_constant(element::i8, weights_shape, ov::test::utils::InputGenerateData(0, 2, 1));
```

Comment:
Can we generate values in wider range? Actually, we can just omit `InputGenerateData` parameter: it will be set automatically in `make_constant` based on the constant precision

Reply:
Removed `InputGenerateData` parameter